docs/markdown/Contributing.md

```diff
@@ -18,6 +18,39 @@ Github](https://github.com/mesonbuild/meson/pulls). This causes them
 to be run through the CI system. All submissions must pass a full CI
 test run before they are even considered for submission.
 
+## Acceptance and merging
+
+The kind of review and acceptance any merge proposal gets depends on
+the changes it contains. All pull requests must be reviewed and
+accepted by someone with commit rights who is not the original
```

Comment:
Just to clarify, this requires review of documentation changes by someone else as well, right?
In my opinion it is reasonable to allow simple non-code changes to be pushed without review,
for complex documentation changes review might be necessary.